### PR TITLE
Use WC Admin package for merchant onboarding tests, fix merchant E2E tests

### LINKED ISSUE #3482: Move to WC Admin tests for the merchant onboarding flows
### Description
Currently, we use the merchant onboarding tests from the `@wocommerece/e2e-utils` in the E2E tests which are being deprecated in favor of using `@woocommerce/admin-e2e-tests`.  This [PR](https://github.com/woocommerce/woocommerce/pull/31235) migrates them in the main WooCommerce repo and some discussion around the same can be checked here: p1638817117032200/1638515774.029000-slack-C026J0AL8NS

### Acceptance criteria

* Move all tests that are being deprecated to use the `@woocommerce/admin-e2e-tests` alternates.

### PR DESCRIPTION
**Pull request by deepakpathania**
Fixes #3482 

#### Changes proposed in this Pull Request
- Move merchant onboarding tests to use `@woocommerce/admin-e2e-tests`, refactor accordingly.
- Fix failing merchant admin E2E tests.

#### Testing instructions
- Refactored tests so try running the updated tests individually, for example:
```
npm run test:e2e-dev tests/e2e/specs/merchant/merchant-orders-manual-capture.spec.js      
```
- Updated tests should pass on this branch and fail on `develop`.

Screenshot 2021-12-13 at 3 38 04 PM

-------------------

- [ ] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)

### CODE REVIEW

File: tests/e2e/specs/merchant/merchant-a-onboarding-task-list.spec.js [hunk 1]
```diff
@@ -1,77 +1,10 @@
...
-	beforeAll( async () => {
-		await merchant.login();
-		await withRestApi.deleteAllShippingZones();
-	} );
-
-	afterAll( async () => {
-		await merchant.logout();
-	} );
-
-	it( 'can dismiss tax setup', async () => {
```
Comment (naman03malhotra) [inline tests/e2e/specs/merchant/merchant-a-onboarding-task-list.spec.js:16]: Does this functionality no longer exist?

**Review by naman03malhotra — APPROVED** [review]
LGTM, Pre approving the pull request left some questions to gain more context.

File: tests/e2e/setup/jest-setup.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const OBSERVED_CONSOLE_MESSAGE_TYPES = {
 const WP_CONTAINER = 'wcp_e2e_wordpress';
 const WP_CLI = `docker run --rm --user xfs --volumes-from ${ WP_CONTAINER } --network container:${ WP_CONTAINER } wordpress:cli`;
 const RESOURCE_TYPES_TO_BLOCK = [ 'image', 'font', 'media', 'other' ];
-const STYLESHEETS_TO_LOAD = [ /\/style.css/, /chunk/, /blocks/ ];
+const STYLESHEETS_TO_LOAD = [ /\/style.css/, /\/menu.css/, /chunk/, /blocks/ ];
```
[the remark this answers is not in the document]
Comment (deepakpathania) [reply tests/e2e/setup/jest-setup.js:35]: No these files were already there, blocking them lead to some functionality not working properly in the E2E test so added them to this whitelist.

File: tests/e2e/specs/merchant/merchant-a-onboarding-task-list.spec.js (hunk 1 above)
Comment (deepakpathania) [reply tests/e2e/specs/merchant/merchant-a-onboarding-task-list.spec.js:16]: These flows are included in the WC Admin test that we have replaced it with.

**Pull request merged**